plutus-core/plutus-core/src/Universe/Core.hs

```diff
@@ -337,6 +342,17 @@ data Kinded uni ta where
 type ValueOf :: (Type -> Type) -> Type -> Type
 data ValueOf uni a = ValueOf !(uni (Esc a)) !a
 
+type HiddenValueOf :: (Type -> Type) -> Type
```

Comment:
name bikeshedding: `SpecializedValueOf`? Since basically the point is to be a specialized version of `ValueOf` for the given universe.